S 68/el 2793 ui updates

chris-brennan opened the pull request:
- Fixed page load column error for the features
- Updated search so that it functions on Micro Focus site as well
- Updated footer logo to handle the text using CSS

ashley-hunter commented on docs/app/components/side-navigation/side-navigation.component.html:
```diff
@@ -5,7 +5,7 @@
                 (ngModelChange)="versionService.setVersion($event)"
                 [btnRadio]="Version.AngularJS">AngularJS</button>
-            <button type="button" class="btn button-toggle-accent"
+            <button type="button" class="btn button-toggle-accent primary-override"
```
Dont think this class is needed here as you should be able to override styles by doing the following:

```css
.side-navigation-container .side-navigation .btn-group .button-toggle-accent {
    // styles go here
}
```

chris-brennan replied:
Sorry I meant to remove this.  I have overridden the styles in ux-aspects-microfocus like you suggested.

ashley-hunter commented on docs/app/data/footer-navigation.json:
```diff
@@ -9,7 +9,8 @@
         }]
     }],
-    "logo": "assets/img/UX-Aspects-Logo.png",
+    "logo": "assets/img/UX-graphic-small.png",
```
Perhaps change this to:

```json
"logo": {
    "image": "assets/img/UX-graphic-small.png",
    "title": "UX Aspects"
}
```

This way the two logo properties are grouped rather than naming one logo and one logoTitle. It also means in future if we were to, for example, make it go to a link when clicked we could just add a link property under logo rather than creating a new `logoLink` property,

donna-marijne commented on docs/app/components/page-footer/page-footer.component.less:
```diff
@@ -40,9 +40,20 @@
+    }
+
+    .page-footer-brand-logo-title {
+        color: #ffffff;
```
Please use the variables as much as possible - it will make it easier to override for the Micro Focus site (which will be black rather than white here, for example).

donna-marijne commented on docs/app/components/side-navigation/side-navigation.component.html:
```diff
@@ -5,7 +5,7 @@
                 (ngModelChange)="versionService.setVersion($event)"
                 [btnRadio]="Version.AngularJS">AngularJS</button>
-            <button type="button" class="btn button-toggle-accent"
+            <button type="button" class="btn button-toggle-accent primary-override"
```
Sorry, this wasn't what I meant for this change. Please bind the button class (.btn-toggle-accent) to an input property on this component. E.g. `[ngClass]="angularButtonClass"`. For the sake of consistency, do this on both buttons. The reason for this is that there are a lot of states and properties defined for buttons in the MF style, and overriding the style on the existing class would involve duplicating a lot style definitions from the quantum stylesheet.

If we used Sass then potentially we could mix in the quantum definitions, but for now I believe that this is the cleanest approach.

chris-brennan commented:
- Removed unused class
- Updated to group logo properties
- Updated to use variables
- Button classes controlled by input property for version toggle

donna-marijne commented on docs/app/components/side-navigation/side-navigation.component.html:
```diff
@@ -1,11 +1,13 @@
          <div class="btn-group version-toggle p-b-sm">
-            <button type="button" class="btn button-toggle-primary"
+            <button type="button" class="btn"
+                [ngClass]="angularButtonClass"
```
Shouldn't this be `angularJsButtonClass`?

chris-brennan commented:
updated to point to angularJsButtonClass

chris-brennan commented:
Resolved conflicts

donna-marijne commented on configs/webpack.docs.dev.config.js:
```diff
@@ -219,7 +219,12 @@ var docsConfig = {
             'process.env': {
                 'ENV': '"development"'
             }
+<<<<<<< HEAD
```
This is a merge file - please resolve. If you have any trouble with merges, let me know and I can describe the workflow.

chris-brennan commented:
resolved conflicts

donna-marijne commented:
@ashley-hunter can you review this? Chris cannot reproduce the above error.

ashley-hunter commented:
Just spoke to him. This branch is outdated and does not have the latest webpack configs, he is rebasing it now as these issues have already been fixed in develop.

donna-marijne commented:
Thanks Ashley.

chris-brennan commented:
not fixed

chris-brennan commented:
That should be it fixed now, thanks.
